src/coreComponents/physicsSolvers/inducedSeismicity/QuasiDynamicEQ.cpp

```diff
@@ -222,19 +230,22 @@ void QuasiDynamicEQ::saveOldStateAndUpdateSlip( ElementSubRegionBase & subRegion
 {
   arrayView1d< real64 > const stateVariable   = subRegion.getField< rateAndState::stateVariable >();
   arrayView1d< real64 > const stateVariable_n = subRegion.getField< rateAndState::stateVariable_n >();
-  arrayView1d< real64 > const slipRate        = subRegion.getField< rateAndState::slipRate >();
-  arrayView1d< real64 > const slipRate_n      = subRegion.getField< rateAndState::slipRate_n >();
-  arrayView1d< real64 > const deltaSlip       = subRegion.getField< rateAndState::deltaSlip >();
-
+  arrayView2d< real64 > const slipRate        = subRegion.getField< rateAndState::slipRate >();
+  arrayView2d< real64 > const slipRate_n      = subRegion.getField< rateAndState::slipRate_n >();
+  arrayView2d< real64 > const deltaSlip       = subRegion.getField< rateAndState::deltaSlip >();
 
-  arrayView1d< real64 > const slip = subRegion.getField< contact::slip >();
+  arrayView2d< real64 > const dispJump = subRegion.getField< contact::dispJump >();
 
   forAll< parallelDevicePolicy<> >( subRegion.size(), [=] GEOS_HOST_DEVICE ( localIndex const k )
   {
-    slipRate_n[k]      = slipRate[k];
-    stateVariable_n[k] = stateVariable[k];
-    deltaSlip[k]       = slipRate[k] * dt;
-    slip[k]            = slip[k] + slipRate[k] * dt;
+    slipRate_n[k][0]    = slipRate[k][0];
+    slipRate_n[k][0]    = slipRate[k][0];
```

Comment:
I think you probably meant to do:
```suggestion
    slipRate_n[k][0]    = slipRate[k][0];
    slipRate_n[k][1]    = slipRate[k][1];
```

In any case, you can use
```suggestion
     LvArray::copy<2>(  slipRate_n, slipRate);
```